.github/workflows/test.yml

```diff
@@ -54,23 +63,23 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
 
     services:
       postgres:
-        image: postgres:11
+        image: bitnami/postgresql:15
```

Comment:
question: why switch to the bitnami version?

in other projects I switched because you can pass additional variables, a la https://github.com/torchbox/wagtail-grapple/blob/main/.github/workflows/ci.yml#L71-L78 to specifically speed up tests

Reply:
No compelling reason, I'll remove it keeping the updated version